src/tds/stream/prepared.rs

```diff
@@ -0,0 +1,84 @@
+use super::ReceivedToken;
```

Comment:
I remember removing this file recently. Maybe a merge conflict gone wrong. We don't really need it and it just causes confusion by existing.